src/js/views/patients/sidebar/action/action-sidebar-activity-views.js

```diff
@@ -129,6 +133,58 @@ const CommentView = View.extend({
   },
 });
 
+const CommentView = View.extend({
+  className: 'u-margin--b-8 qa-activity-item',
+  modelEvents: {
+    'change:editing': 'render',
+  },
+  childViewTriggers: {
+    'click:edit': 'click:edit',
+    'cancel:comment': 'cancel:edit',
+    'post:comment': 'save:comment',
+    'delete:comment': 'delete:comment',
+  },
+  template: hbs`
+    <div data-display-region></div>
+    <div class="sidebar_activity--comment-form" data-form-region></div>
+  `,
+  regions: {
+    display: '[data-display-region]',
+    form: '[data-form-region]',
+  },
+  onRender() {
+    if (this.model.get('editing')) {
+      this.showFormView();
+      return;
+    }
+
+    this.showReadOnlyView();
+  },
+  showReadOnlyView() {
+    this.getRegion('form').empty();
+    this.showChildView('display', new CommentDisplayView({ model: this.model }));
+  },
+  showFormView() {
+    this.getRegion('display').empty();
+    this.showChildView('form', new CommentFormView({ model: this.model.clone() }));
+  },
+  onClickEdit() {
+    this.model.set('editing', true);
+  },
+  onCancelEdit() {
+    this.model.set('editing', false);
+  },
+  onSaveComment({ model }) {
+    this.model.save({
+      message: model.attributes.message,
+    });
+    this.model.set({
+      'editing': false,
```

Comment:
quotes aren't needed here.  Can we add the `edited_at` to the `save` so there's just a single change event?  If we ever do anything off the change event in the future, the `edited_at` won't be updated.  hopefully the server would just ignore it?  I believe that's true.  Also it needs to be `moment.utc().format()`